### PR TITLE
Change param of to hexadecimal  of Eth fee history to match spec

### PR DESCRIPTION
**Pull request by gfukushima**
## PR description
Change param type to accept hexadecimal and match the spec: https://ethereum.github.io/execution-apis/api-documentation/

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if
    [updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by jframe — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,12 @@
 
 ### Breaking Changes
 - Add a new CLI option to limit the number of requests in a single RPC batch request. Default=1 [#4965](https://github.com/hyperledger/besu/pull/4965)
-- Changed JsonRpc http service to return the error -32602 (Invalid params) with a 200 http status code
+- Change JsonRpc http service to return the error -32602 (Invalid params) with a 200 http status code
 - Besu requires minimum Java 17 and up to build and run [#3320](https://github.com/hyperledger/besu/issues/3320)
 - PKCS11 with nss module (PKCS11 based HSM can be used in DevP2P TLS and QBFT PKI) does not work with RSA keys 
 in Java 17. SoftHSM is tested manually and working. (Other PKCS11 HSM are not tested). The relevant unit and acceptance 
 tests are updated to use EC private keys instead of RSA keys.
+- Change eth_feeHistory parameter `blockCount` to accept hexadecimal string #5047
```
Comment (macfarla) [inline CHANGELOG.md:12]:
```suggestion
- Change eth_feeHistory parameter `blockCount` to accept hexadecimal string [#5047](https://github.com/hyperledger/besu/issues/5047)
```

Comment (macfarla) [reply CHANGELOG.md:12]: I made the above suggestion - but I think your way also works because we're in GitHub and it's much simpler...

Comment (macfarla) [reply CHANGELOG.md:12]: maybe say why it's a breaking change - accept hexadecimal string (was accepting plain integer)

**Review by macfarla — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (gfukushima) [reply CHANGELOG.md]: Done, ty

**Pull request merged**